scheduler/utilization_based_host_allocator.go

```diff
@@ -64,21 +63,31 @@ func UtilizationBasedHostAllocator(ctx context.Context, hostAllocatorData HostAl
 		}
 
 		// add up total number of hosts needed for all groups
-		nNewHosts += n
+		nNewHostsRequired += n
+	}
+
+	// Will at least distro.PlannerSettings.MinimumHosts be running once nNewHostsRequired are up and running?
+	minimumHostsThreshold := distro.PlannerSettings.MinimumHosts
+	nExistingAndRequiredHosts := nExistingHosts + nNewHostsRequired
+	nAdditionalHostsToMeetMinimum := 0
+	if nExistingAndRequiredHosts < minimumHostsThreshold {
+		nAdditionalHostsToMeetMinimum = minimumHostsThreshold - nExistingAndRequiredHosts
 	}
-	calcNumHostsRuntime := time.Since(calcNumHostsBeginsAt)
 
 	grip.Info(message.Fields{
-		"runner":         RunnerName,
-		"distro":         distro.Id,
-		"num_new_hosts":  nNewHosts,
-		"message":        "requesting new hosts",
-		"group_dur_secs": taskGroupingRuntime.Seconds(),
-		"group_num":      len(taskGroupDatas),
-		"calc_dur_secs":  calcNumHostsRuntime.Seconds(),
+		"runner":                               RunnerName,
+		"message":                              "requesting new hosts",
+		"distro":                               distro.Id,
+		"group_dur_secs":                       taskGroupingDuration.Seconds(),
+		"group_num":                            len(taskGroupDatas),
+		"minimum_hosts_for_distro":             minimumHostsThreshold,
+		"num_existing_hosts":                   nExistingHosts,
+		"num_new_hosts_required:":              nNewHostsRequired,
+		"num_additional_hosts_to_meet_minimum": nAdditionalHostsToMeetMinimum,
+		"total_new_hosts_to_request":           nNewHostsRequired + nAdditionalHostsToMeetMinimum,
```

Comment:
should we save this into a variable, given that we perform the addition twice?

Reply:
Done.